app-modules/form/resources/views/blocks/submissions/student-email.blade.php

```diff
@@ -0,0 +1,36 @@
+<x-form::blocks.field-wrapper
+    class="py-3"
+    :$label
+    :$isRequired
+>
+    @if (filled($response ?? null))
+        <div class="not-prose flex flex-wrap items-center gap-3">
+            <span>{{ $response ?? null }}</span>
+            @if ($authorType === \Assist\AssistDataModel\Models\Student::class)
+                <a
+                    href="{{ \Assist\AssistDataModel\Filament\Resources\StudentResource::getUrl('view', ['record' => $authorKey]) }}"
+                    target="_blank"
+                >
+                    <x-filament::badge color="success">
+                        Student
+                    </x-filament::badge>
+                </a>
+            @elseif ($authorType === \Assist\Prospect\Models\Prospect::class)
+                <a
+                    href="{{ \Assist\Prospect\Filament\Resources\ProspectResource::getUrl('view', ['record' => $authorKey]) }}"
```

Comment:
A little nit, but I think it is cleaner if we import all these classes in use statements in a `@php` block at the top of the file like we do elsewhere in the app. What do you think?

Reply:
Done